samples/nodejs_simple_sample/src/gateway_sample_lin.json

```diff
@@ -1,11 +1,20 @@
 {
+    "loaders": [
```

Comment:
This was given to me as is by Raj. There was conversation around this at scrum that sounded like a firm "maybe".